src/main/ml-modules/root/com.marklogic.smart-mastering/collector.xqy

```diff
@@ -0,0 +1,33 @@
+xquery version "1.0-ml";
```

Comment:
Added a URI collector for use with sm-match-and-merge endpoint that will cover most cases.